Disable using tr1/unordered_map in dyntypes.h.  This version of GCC
from configure doesn't support it, so fall back to ext/hash_map.
This works around a problem on some systems, eg, module at05 on biou,
that are version >= 4.3 but don't have unordered map.


diff -Naurb SymtabAPI-8.0.orig/dynutil/h/dyntypes.h SymtabAPI-8.0/dynutil/h/dyntypes.h
--- SymtabAPI-8.0.orig/dynutil/h/dyntypes.h	2012-11-20 15:32:26.000000000 -0600
+++ SymtabAPI-8.0/dynutil/h/dyntypes.h	2012-11-29 18:19:54.000000000 -0600
@@ -55,8 +55,9 @@
   #include <functional>
   #define DECLTHROW(x) throw(x)
   //***************** GCC ***********************
-   #if (__GNUC__ > 4) || \
-      (__GNUC__ == 4 && __GNUC_MINOR__ >= 3)
+  // DISABLE_TR1_MAP: This version of GCC doesn't support
+  // tr1/unordered_map, so we have to fall back on ext/hash_map.
+   #if 0
       //**************** GCC >= 4.3.0 ***********
       #include <tr1/unordered_set>
       #include <tr1/unordered_map>
